packages/admin/cms-admin/src/form/file/upload.ts

```diff
@@ -16,6 +17,9 @@ export function upload<ResponseData>(
     if (data.folderId !== undefined) {
         formData.append("folderId", data.folderId);
     }
+    if (data.scope !== undefined && Object.keys(data.scope).length > 0) {
```

Comment:
I need this check for the object's keys because the DAM scope is `{}` by default, which the resolvers ignore, but the upload controller doesn't. Is this okay, or should I change the controller?

Reply:
It would sound logical to me if controller/resolver behaves as similar as possible.

Reply:
Done: 12cf38a4cdd9a81ec73bc5c28ed8160761544942